dockstore-webservice/src/main/java/io/dockstore/webservice/Constants.java

```diff
@@ -38,4 +38,14 @@ public final class Constants {
     private Constants() {
         // not called
     }
+
+    public static final class NamedQueries {
+        public static final String ENTRY_GET_EXECUTION_METRIC_PARTNERS = "Entry.getExecutionMetricsPartners";
```

Comment:
This location seems unexpected, why not put it in Entry?

Reply:
For some reason, I had it in my head that you couldn't refer to a constant from the same file/class in an class-level annotation, but I was wrong. I'll move it.